index.html

```diff
@@ -565,30 +537,26 @@
           </li>
           <li data-link-for="PaymentDetailsInit">Process the total:
             <ol>
-              <li>If <var>details</var>.<a>total</a>.<a data-lt=
-              "PaymentItem.amount">amount</a>.<a data-lt=
-              "PaymentCurrencyAmount.value">value</a> is not a <a>valid decimal
-              monetary value</a>, then <a>throw</a> a <a>TypeError</a>;
-              optionally informing the developer that the value is invalid.
-              </li>
-              <li>If the first <a>code point</a> of
+              <li>Let <var>total</var> be
               <var>details</var>.<a>total</a>.<a data-lt=
-              "PaymentItem.amount">amount</a>.<a data-lt=
-              "PaymentCurrencyAmount.value">value</a> is U+002D HYPHEN-MINUS,
-              then <a>throw</a> a <a>TypeError</a>, optionally informing the
-              developer that the total can't be negative.
+              "PaymentItem.amount">amount</a>.
+              </li>
+              <li>
+                <a>Check and canonicalize <var>total</var></a>. Rethrow any
```

Comment:
I'm a little weirded out by how the variable name is both a variable name and part of the algorithm name. IMO "[Check and canonicalize the total] _total_" would be better.